[Docs][Windows] Update Windows doc

thexai opened the pull request:
## Description
<!--- Provide a general summary of your change in the Pull Request title above -->
<!--- Describe your change in detail here. -->
- Remove references to Visual Studio 2019.
- Remove references to UWP-ARM (32bit).
- Add references to ARM64  (desktop).
- Add CMake example for generate ARM64 project (desktop).
- Some misc. updates.

## Motivation and context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->
- ARM64 now is supported.  --> nightlies [here](https://mirrors.kodi.tv/nightlies/windows/winarm64/master/)
- UWP ARM (32bit) is not supported anymore in the latest Windows SDK's (even previous SDK's with updates).

## How has this been tested?
<!--- Please describe in detail how you tested your change. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## What is the effect on users?
<!--- Summarize the effect of this change on Kodi end-users. -->
<!--- If the PR does not have a noticeable impact (e.g., if it only changes documentation), -->
<!--- just leave it empty. Put in more detail the bigger the impact is. -->
<!--- This section may be used for automatic creation of release notes. -->

## Screenshots (if appropriate):

## Types of change
<!--- What type of change does your code introduce? Put an `x` with no space in all the boxes that apply like this: [X] -->
- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [X] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **Student submission** (PR was done for educational purposes and will be treated as such)
- [ ] **None of the above** (please explain below)

## Checklist:
<!--- Go over all the following points, and put an `X` with no space in all the boxes that apply like this: [X] -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [X] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [X] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

thexai committed:
[Docs][Windows] Update Windows doc

garbear commented on docs/README.Windows.md:
````diff
@@ -220,6 +220,11 @@ Or configure build for 32bit:
 
+Or configure build for ARM 64bit:
+```
+cmake -G "Visual Studio 17 2022" -A arm64 -T host=x64 %userprofile%\kodi
````
I build WIN-ARM64 on WIndows arm64 (parallels vm on Macbook M2). We should probably remove all `-T` flags

garbear commented:
We also remove support for Windows 8.1 in this doc, right?

scott967 commented:
From a "typical" x64 build person LGTM

thexai commented:
In the latest Visual Studio 2022 updates "Universal Windows Platform tools" location has changed as Optional component under new "WinUI application development" section:

<img width="1238" height="628" alt="VS_2022" src="https://github.com/user-attachments/assets/6dbcbe62-b31c-4e6d-bacf-e7f7fdfb6520" />

Changed because now is not very intuitive found Universal Windows Platform tools...

Also changed cmake command to generate ARM64 removing `-T host` parameter (only in this case because it is more likely to be used by those who have an native ARM64 platform and also works for build from x64 platform).

MikeSiLVO commented:
Is there a need for ```make-addons.bat peripheral.joystick```?

I always build the installer and this step hasn't worked nor does it seem to be needed for a long while.

Thanks :)
